components/clipboard/clipboard.js

```diff
@@ -0,0 +1,36 @@
+import clipboardStyle from './clipboard.module.css';
+import {Component} from "react";
+
+class Clipboard extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  copyToClipboard = str => {
+    const temp = document.createElement('textarea');
```

Comment:
as this is a side project, I want this project to use cutting edge stuff even if browser support is low. So I suggest use the new 

```js
await navigator.clipboard.writeText(textToCopy);
```

to copy the code.

More details here-
https://dev.to/mornir/-how-to-easily-copy-text-to-clipboard-a1a